src/21-pubsub/index.md

```diff
@@ -24,25 +24,23 @@ collection: posts
 ---
 
 To make IPFS fast for datacenters, local area networks, and large p2p
-applications, we need a fast publish subscribe system. [Publish
-Subscribe](https://en.wikipedia.org/wiki/Publish%E2%80%93subscribe_pattern),
-called 'pubsub' for short, is a mechanism for 'publishers' to distribute
-messages quickly and directly to 'subscribers' that are interested in them.
-
-Before going down the road of implementing a new pubsub system, we did a fairly
-extensive review of as much pubsub literature and research as we could find. We
-are planning on doing a writeup on this process soon, but we've been pretty
-busy implementing other things and haven't had time to get to that yet.
+applications, we need a fast publish-subscribe system. [Publish-Subscribe](https://en.wikipedia.org/wiki/Publish%E2%80%93subscribe_pattern),
+called 'pubsub' for short, is a pattern often used to handle events in
+large-scale networks. 'Publishers' send messages classified by topic or content and
+'subscribers' receive only the messages they are interested in, all without direct
+connections between publishers and subscribers. This approach offers much greater
+network scalability and flexibility.
 
 We have recently merged a simple, experimental pubsub implementation into go-ipfs.
-The implementation is experimental, and is very far from the performance and
-security levels we expect from our long-term target. But already, this feature
-does allow for some very useful and fun new applications to be developed. In
-this post I will explain how to get started using `ipfs pubsub` and give some
-example applications that can and have been built using it.
-
-First off, to enable the pubsub code, make sure you're running a recent version
-of go-ipfs, or try out the pre-built floodsub binaries up on [the distributions
+The implementation is very far from the performance and security levels we expect
+from our long-term target. But already, this feature allows for some very useful and interesting new applications.
+
+In this post, I will explain how to get started using `ipfs pubsub` and share some
+current and future examples of applications.
+
+## Getting started with ipfs pubsub
+First, you'll need to enable the pubsub code. Make sure you're running a recent version
+of go-ipfs, or try out the pre-built floodsub binaries up on the [distributions
```

Comment:
Can remove the mention of the floodsub binaries since its merged into recent go-ipfs version